examples/Azure_IoT_Central_ESP32/iot_configs.h

```diff
@@ -5,10 +5,54 @@
 #define IOT_CONFIG_WIFI_SSID              "SSID"
 #define IOT_CONFIG_WIFI_PASSWORD          "PWD"
 
+// Define if using device x509 certificates
+// #define IOT_CONFIG_USE_X509_CERT
+
+#ifdef IOT_CONFIG_USE_X509_CERT
+
+/*
+
+Device certificate should be of the form:
+
+"-----BEGIN CERTIFICATE-----\r\n" \
+"MIIBJDCBywIUfeHrebBVa2eZAbouBgACp9R3BncwCgYIKoZIzj0EAwIwETEPMA0G\r\n" \
+"A1UEAwwGRFBTIENBMB4XDTIyMDMyMjazMTAzN1oXDTIzMDMyMjIzMTAzN1owGTEX\r\n" \
+"MBUGA1UEAwwOY29udG9zby1kZXZpY2UwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNC\r\n" \
+.......
+"YmGzdaHTb6P1W+p+jmc+jJn1MAoGCXqGSM49BAMCA0gAMEUCIEnbEMsAdGFroMwl\r\n" \
+"vTfQahwsxN3xink9z1gtirrjQlqDAiEAyU+6TUJcG6d9JF+uJqsLFpsbbF3IzGAw\r\n" \
+"yC+koNRC0MU=\r\n" \
+"-----END CERTIFICATE-----"
+
+*/
+#define IOT_CONFIG_DEVICE_CERT "Device Certificate"
+
+/*
+
+Device certificate private key should be of the form:
+
+"-----BEGIN EC PRIVATE KEY-----\r\n" \
+"MHcCAQEEIKGXkMMiO9D7jYpUjUGTBn7gGzeKPeNzCP83wbfQfLd9obAoGCCqGSM49\r\n" \
+"AwEHoUQDQgAEU6nQoYbjgJvBwaeD6MyAYmOSDg0QhEdyyV337qrlIbDEKvFsn1El\r\n" \
+"yRabc4dNp2Jhs3Xh02+j9Vvqfo5nPoyZ9Q==\r\n" \
+"-----END EC PRIVATE KEY-----"
+
+Note the type of key may different in your case. Such as BEGIN PRIVATE KEY
+or BEGIN RSA PRIVATE KEY.
+
+*/
+#define IOT_CONFIG_DEVICE_CERT_PRIVATE_KEY "Device Certificate Private Key"
+
+#endif // IOT_CONFIG_USE_X509_CERT
+
 // Azure IoT Central
 #define DPS_ID_SCOPE                      "ID Scope"
```

Comment:
Instead of these being "ID Scope", etc we should have some examples - e.g. "0ne12345" for IDScope.

Reply:
I rather put that in the comment above the define, as an example.